backend/composer/models.py

```diff
@@ -240,18 +241,66 @@ class Meta:
         verbose_name_plural = "Anatomical Entities"
 
 
-class Layer(AnatomicalEntityMeta):
-    ...
+class Layer(models.Model):
+    layer_id = models.BigAutoField(primary_key=True, auto_created=True)
+    ae_meta = models.ForeignKey(AnatomicalEntityMeta, on_delete=models.CASCADE, related_name='layer_meta', null=True)
+
+    def __str__(self):
+        return self.ae_meta.name
+    
+    def clean(self):
```

Comment:
suggestion (non-blocking): Clean and save seem to be doing the same checks. It might make sense to replace the save checks with a call to clean.